ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1115,8 +1116,8 @@ def test_method(
         for v, d in zip(init_flags.as_variable, init_input_dtypes)
     ]
 
-    # Create Args
-    args_constructor, kwargs_constructor, *_ = create_args_kwargs(
+    # Save original constructor data for inplace operations
+    con_data = OrderedDict(
```

Comment:
Is it necessary to create an OrderedDict here? can't we just copy the original args_constructor and kwargs_constructor?

Reply:
https://github.com/unifyai/ivy/blob/74d08c39bb607eb09b020146388e7c0bfceceb3b/ivy_tests/test_ivy/helpers/function_testing.py#L1219-L1222

args_constructor and kwargs_constructor may not be ordered (I don't think our Container is ordered in the early python3s? It only inherits from Dict), and later we may have to pass the arguments in the correct order to another process.